src/main/java/uk/gov/ons/census/action/model/entity/ActionRule.java

```diff
@@ -33,7 +26,6 @@
 
   @Column private Boolean hasTriggered;
 
-  @Type(type = "jsonb")
-  @Column(columnDefinition = "jsonb", nullable = false)
-  private Map<String, List<String>> classifiers;
+  @Column(nullable = false, length = 100000)
+  private String userDefinedWhereClause;
```

Comment:
as mentioned on other ticket, it's not 'user defined'... it's a classifier [fragment].